src/components/login-form.js

```diff
@@ -1,14 +1,110 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteForm from './vote-form';
 
 export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
+	const [logForm, setLogForm] = useState({ email: '', password: '' });
+	const [isloged, setloged] = useState(false);
+
+	const erasePW = () => {
+		document.getElementById('password').value = '';
+	};
+
+	const eraseEmail = () => {
+		document.getElementById('email').value = '';
+	};
+
+	const handleFormChange = (e) => {
+		setLogForm({ ...logForm, [e.target.id]: e.target.value });
+	};
+
+	const handleSubmit = (logForm) => {
+		const { email, password } = logForm;
+		if (email === '' || password === '') {
+			alert('모든 항목을 입력해주세요!');
+			return false;
+		} else {
+			axios
+				.post(process.env.API_HOST + '/auth/signin/', logForm)
+				.then(function (response) {
+					console.log(response);
+					setloged(true);
+					alert('로그인 성공!');
+				})
+				.catch(function (error) {
+					if (error.response.status === 404) {
+						console.log('unauthorized, logging out ...');
+						alert('이메일이 존재하지 않습니다.');
+						eraseEmail();
+						erasePW();
+					} else if (error.response.status === 422) {
+						alert('비밀번호가 일치하지 않습니다!');
+						erasePW();
+					}
+					return Promise.reject(error.response);
+				});
+		}
+	};
+
+	return (
+		<div>
+			{!isloged && (
+				<Wrapper>
+					<Title>로그인</Title>
+					<Row>
+						<Label>EMAIL</Label>
+						<Input type="text" id="email" onChange={handleFormChange}></Input>
```

Comment:
id 사용 좋네요~! name으로도 바꿔보세요!
```suggestion
						<Input type="text" name="email" onChange={handleFormChange}></Input>
```